tiers/checks.go

```diff
@@ -133,7 +147,7 @@ func (a *advancedTierChecks) limitRangeByType(nsType string) namespaceObjectsChe
 type teamTierChecks struct {
 }
 
-func (a *teamTierChecks) GetNamespaceObjectChecks(nsType string) []namespaceObjectsCheck {
+func (a *teamTierChecks) GetNamespaceObjectChecks(templateRef string) []namespaceObjectsCheck {
```

Comment:
the parameter should be `nsType` and not `templateRef`
```suggestion
func (a *teamTierChecks) GetNamespaceObjectChecks(nsType string) []namespaceObjectsCheck {
```